.github/workflows/validate-json.yaml

```diff
@@ -0,0 +1,32 @@
+name: Validate emojibase.json
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
Any reason to not use `ubuntu-latest` here?

Reply:
No, was a copy/paste :)